network_runner/resources/inventory.py

```diff
@@ -0,0 +1,116 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+from network_runner.types.objects import Object
+from network_runner.types.attrs import Attribute
+from network_runner.types.attrs import Container
+from network_runner.types.attrs import SERIALIZE_WHEN_PRESENT
+from network_runner.types.attrs import SERIALIZE_WHEN_NEVER
+from network_runner.types.validators import ChoiceValidator
+
+
+NETWORK_OS_VALIDATOR = ChoiceValidator(
+    choices=('cumulus', 'dellos10', 'eos', 'junos', 'nxos', 'openvswitch')
+)
+
+
+class Host(Object):
+
+    name = Attribute(
+        required=True,
+        serialize_when=SERIALIZE_WHEN_NEVER
+    )
+
+    ansible_host = Attribute(
+        serialize_when=SERIALIZE_WHEN_PRESENT
+    )
+
+    ansible_user = Attribute(
+        serialize_when=SERIALIZE_WHEN_PRESENT
+    )
+
+    ansible_password = Attribute(
```

Comment:
We had trouble passing ansible_password instead of ansible_ssh_pass. The ML2 driver assumes ssh_pass.
Could we keep it ssh_pass or handle both we don't break the ml2 driver? We'll probably need to test the transition outside of this patch to make sure things don't break.

Reply:
Just pushed a change that allows attributes to define one or more aliases.  Setting one will set all.  Let me know if that addresses your concern.